puppet/zulip/files/cron.d/soft-deactivate-users

```diff
@@ -0,0 +1,6 @@
+SHELL=/bin/bash
+PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+USER=zulip
+
+# Cron job to soft deactivate long term idle users weekly.
+0 5 * * 7 zulip /home/zulip/deployments/current/manage.py soft_activate_deactivate_users -d --inactive-for 30 > /dev/null
```

Comment:
Also, do we need the `--inactive-for` argument here?  Seems like we could just have it use the default.